Makefile

```diff
@@ -132,10 +132,12 @@ define PROJECT_ENV
 endef
 
 LOCAL_DEPS = sasl mnesia os_mon inets
-BUILD_DEPS = rabbitmq_cli
+BUILD_DEPS = rabbitmq_cli syslog
 DEPS = ranch lager rabbit_common
 TEST_DEPS = rabbitmq_ct_helpers rabbitmq_ct_client_helpers amqp_client meck proper
 
+dep_syslog = git https://github.com/schlagert/syslog master
```

Comment:
Could you please pin the version, so the library is the same for everyone?

The latest tag is `3.4.2`.